pkg/controller/core.oam.dev/v1alpha2/application/suite_test.go

```diff
@@ -217,9 +218,7 @@ func (f *FakeRecorder) Event(object runtime.Object, eventtype, reason, message s
 }
 
 func (f *FakeRecorder) Eventf(object runtime.Object, eventtype, reason, messageFmt string, args ...interface{}) {
-	if f.Events != nil {
-		f.Events <- fmt.Sprintf(eventtype+" "+reason+" "+messageFmt, args...)
-	}
+	f.Event(object, eventtype, reason, messageFmt)
```

Comment:
You can delete the `f.Events` in FakeRecorder as it's not used

Reply:
the logic have used `f.Events`. This change is for fix failed event related test